appengine/metadata/standard/server.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START appengine_metadata]
+const express = require('express');
+const request = require('got');
+
+const app = express();
+app.enable('trust proxy');
+
+const METADATA_NETWORK_INTERFACE_URL = 'http://metadata/computeMetadata/v1/' +
```

Comment:
Nit: combine this into one string (and ignore line length limits, or split onto multiple lines using backticks)?

Reply:
Done :)